requirements.in

```diff
@@ -3,19 +3,19 @@
 # License: BSD
 # Upstream url: http://github.com/mitsuhiko/flask/
 # Use: For API.
-Flask==1.1.1
+Flask
```

Comment:
are license comments required? if not, thoughts on removing them as well?

Reply:
I think it's silly to check them in like this (would rather use a tool to check it automatically), but left them in because having them does hint that contributors should pay attention to licenses, which is important for ensuring this OSS code is actually reusable